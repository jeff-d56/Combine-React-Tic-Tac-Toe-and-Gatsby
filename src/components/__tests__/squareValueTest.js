import React from "react";
import Square from "../TicTacToe/square";
import renderer from "react-test-renderer"

var testBoard = 0;
describe('test if square displays correct value', () => {
	it("renders correctly", () => {
    
    const tree = renderer.create(<Square
        value={0}
        onClick={() => this.props.onClick(i)}
      />).toJSON()
    expect(tree).toMatchSnapshot()
  })
  
});



