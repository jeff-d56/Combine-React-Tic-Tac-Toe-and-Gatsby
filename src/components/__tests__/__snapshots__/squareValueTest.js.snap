// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`test if square displays correct value renders correctly 1`] = `
<button
  className="square"
  onClick={[Function]}
>
  0
</button>
`;
